editoast/src/schema/operation/mod.rs

```diff
@@ -58,9 +58,9 @@ impl Operation {
 #[editoast_error(base_id = "operation")]
 enum OperationError {
     // To modify
-    #[error("Object '{0}', could not be found")]
+    #[error("Object '{0}', could not be found in the infrastructure '{1}'")]
     #[editoast_error(status = 404)]
-    ObjectNotFound(String),
+    ObjectNotFound(String, i64),
```

Comment:
We could enhance the error payload.
```suggestion
    ObjectNotFound{obj_id: String, infra_id: i64},
```